utils/constants.ts

```diff
@@ -1,4 +1,9 @@
 export const deploymentAddresses: Record<string, Record<string, string>> = {
+  hardhat: {
+    CONTRACT_REGISTRY_ADDRESS: "0x0000000000000000000000000000000000000000",
+    BCT_ADDRESS: "0x0000000000000000000000000000000000000000",
+    NCT_ADDRESS: "0x0000000000000000000000000000000000000000",
+  },
```

Comment:
it took me a while to realize that you don't need these because the tests are run straight on Mumbai. Maybe a comment clarifying why all these contracts are zeroed would help future new-joiners like me understanding the rationale behind it, because on the surface it looks a bit odd

Reply:
This is needed in order to deploy the contract in the Hardhat network. Added a comment.